lighthouserc.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  ci: {
+    collect: {
+      url: ['http://localhost:1313/'],
+      startServerCommand: 'hugo server',
+    },
+    upload: {
+      target: 'lhci',
+      serverBaseUrl: 'https://lighthouse.hedgedoc.net',
```

Comment:
Please don't pin this to hedgedoc.net. Ask @SISheogorath for a subdomain on hedgedoc.org

Reply:
Good point, I just wanted to avoid sending everything to Google's servers and have some running config.
But I agree, a hedgedoc.org sub-domain would be even better.